language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -82,79 +115,154 @@ mergePackageMap ps = foldM merge Map.empty ps
                     pure (Just (mbOldPkgName <|> mbPkgName, pkg))
         Map.alterF update pkgId pkgs
 
+-- Write packages for all the DALFs in all the DARs.
 main :: IO ()
 main = do
     opts@Options{..} <- execParser optionsParserInfo
+    let damlTypesVersion = fromMaybe "0.0.0" optDamlTypesVersion
+       -- The '@daml/types' version that the packages we write should depend on.
+    let packageVersion   = fromMaybe "0.0.0" optPackageVersion
```

Comment:
As mentioned elsewhere, I believe this really should be the version of the DALF not the SDK version. Fine to leave it for this PR (although I would like to see the logic move here instead of in `daml-helper`) but maybe add a TODO.

This also means that you need different versions for each package so you need to change the CLI flags to allow passing a version for a specific package.

Reply:
Actually removing the package argument from the command line entirely. Noted regarding the rest but let's get @hurryabit to chime in with his approval on that strategy too.